scripts/build-profiles/stencil-component/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@okta/build-profiles.stencil-component",
+  "description": "Tools to build, run, and test a stencil component",
+  "private": true,
+  "version": "0.0.0",
+  "bin": {
+    "stencil-component": "./bin/stencil-component.js"
+  },
+  "dependencies": {
+    "@okta/scripts.logger": "0.1.0-beta.4322.g65a2415",
+    "@stencil/core": "2.3.0",
+    "@stencil/sass": "1.1.1",
+    "@stencil/utils": "^0.0.5",
+    "@storybook/addon-a11y": "^6.0.0-beta.33",
+    "@storybook/addon-actions": "^6.0.0-beta.33",
+    "@storybook/addon-centered": "^6.0.0-alpha.30",
```

Comment:
```suggestion
```

This dep is not necessary as storybook provides a way to do this out of the box

Here is a reference implementation from my branch: https://github.com/okta/odyssey/blob/storybook/packages/storybook/stories/Tooltip.stories.js#L8